engine/XcodeUnitTestEngine.php

```diff
@@ -43,16 +44,6 @@ public function shouldEchoTestResults() {
     return false; // i.e. this engine does not output its own results.
   }
 
-  private function shouldGenerateCoverage() {
-    // getEnableCoverage return value meanings:
-    // false: User passed --no-coverage, explicitly disabling coverage.
-    // null:  User did not pass any coverage flags. Coverage should generally be enabled if
-    //        available.
-    // true:  User passed --coverage.
-    // https://secure.phabricator.com/T10561
-    return $this->getEnableCoverage() !== false;
```

Comment:
The logic and comment here are important to maintain. Can you keep this function in place, or did it need to be removed?